pages/donate.html

```diff
@@ -1,7 +1,14 @@
 <section class="inner">
-  <h2>Donate</h2>
 
-  <p>
-    Placeholder content for donate page.
-  </p>
+	<h2>Donate to The Oppia Foundation</h2>
+	<p>
+	With your support, the Oppia Foundation can continue its mission to build personal, responsive lessons by uniting a global community of educators.
```

Comment:
Should be indented by 2 from above, since this text is a "child" of the parent p tag. Ditto elsewhere.

Reply:
done